lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE FunctionalDependencies #-}
```

Comment:
:thinking: I haven't seen yet why this was introduced, but we can usually solve same problems with `TypeFamilies` which IMO gives more readable code and compilation errors.

Reply:
Redundant - dropping...